src/index.js

```diff
@@ -24,17 +24,13 @@ const slackBot = new SlackBot({
   token: process.env.SLACK_TOKEN,
 });
 
-const rules = new models.Rules();
-
-// TODO: this should obs come from some JSON config or similar
-const expression = new RegExp(/guys/);
-const reaction = new reactions.ReactionDirectMessage('Guys is not cool');
-rules.add(new models.Rule(expression, reaction));
+const rulesJson = JSON.parse(fs.readFileSync('./config.json', { encoding: 'utf8' }));
```

Comment:
Interesting! The problem is that `require` calls are relative to the file making the call, whereas `fs` calls are relative to the current working directory.

This change would break our ability to run `node src/index` as we currently do. We would need to also change how we layout the files when the app runs within Docker (is different, since we don't pack up the test files) or only run the app within Docker, even when doing local development.

All in all doable, but I also prefer it to be a bit more in-your-face than having to know what `require` does behind the scenes.